.bumpversion.cfg

```
@@ -1,6 +1,6 @@
 [bumpversion]
 commit = True
 tag = True
-current_version = 5.14.1
+current_version = 5.14.2
```

question (non-blocking): maybe we should use a distinct versioning? 
```suggestion
current_version = 5.14.1.0
```

If we have to apply a security or anything we can just make it `<nylas-version>.<our-patch>`

* `5.14.1` -> `5.14.1.0` -> `5.14.1.1` (just in case we need to)
* `5.14.2` -> `5.14.2.0`

Just a suggestion, non-blocking